src/main/kotlin/pl/jutupe/ktor_rabbitmq/RabbitMQ.kt

```diff
@@ -35,6 +43,12 @@ class RabbitMQ(
     fun <T> serialize(body: T): ByteArray =
         configuration.serializeBlock.invoke(body!!)
 
+    override fun close() {
```

Comment:
shouldn't it close channel and connection?

Reply:
As per Rabbit client's ConnectionFactory:

> Set the executor to use for connection shutdown. All connections that use this executor share it

I think that'll make this plugin's graceful shutdown to work as expected.